README.md

```diff
@@ -39,98 +41,165 @@ Current stable release is able to persist Orion context data in:
+[Top](#top)
 
-    POST http://localhost:1028/notify
-    Content-Length: 492
-    User-Agent: orion/0.9.0
-    Host: localhost:1028
-    Accept: application/xml, application/json
-    Content-Type: application/json
-    Fiware-Service: my-company-name
-    Fiware-ServicePath: /workingrooms/floor4 
-    
+###<a name="section3.1"></a>Subscription to the NGSI-like source
+Cygnus takes advantage of the subscription-notification mechanism of NGSI. Specifically, Cygnus needs to be notified each time certain entity's attributes change, and in order to do that, Cygnus must subscribe to those entity's attribute changes.
+
+As long as the typical NGSI-like source is Orion Context Broker, you can make a subscription about the example NGSI entity ('car1' of type 'car') by using the `curl` command in this [way](https://forge.fi-ware.eu/plugins/mediawiki/wiki/fiware/index.php/Publish/Subscribe_Broker_-_Orion_Context_Broker_-_User_and_Programmers_Guide#ONCHANGE) (assuming Orion runs in localhost and listens on the TCP/1026 port):
+
+    (curl localhost:1026/v1/subscribeContext -s -S --header 'Content-Type: application/json' --header 'Accept: application/json' -d @- | python -mjson.tool) <<EOF
     {
-      "subscriptionId" : "51c0ac9ed714fb3b37d7d5a8",
-      "originator" : "localhost",
-      "contextResponses" : [
-        {
-          "contextElement" : {
-            "attributes" : [
-              {
-                "name" : "temperature",
-                "type" : "centigrade",
-                "value" : "26.5",
-                "metadatas": [
-                  {
-                     "name": "ID",
-                     "type": "string",
-                     "value": "ground"
-                  }
+        "entities": [
+            {
+                "type": "car",
+                "isPattern": "false",
+                "id": "car1"
+            }
+        ],
+        "attributes": [
+            "speed",
+            "oil_level"
+        ],
+        "reference": "http://localhost:5050/notify",
+        "duration": "P1M",
+        "notifyConditions": [
+            {
+                "type": "ONCHANGE",
+                "condValues": [
+                    "speed"
                 ]
-              }
-            ],
-            "type" : "Room",
-            "isPattern" : "false",
-            "id" : "Room1"
-          },
-          "statusCode" : {
-            "code" : "200",
-            "reasonPhrase" : "OK"
-          }
-        }
-      ]
+            }
+        ],
+        "throttling": "PT1S"
     }
+    EOF
+
+Which means: <i>Each time the the 'car1' entity, of type 'car', changes its value of 'speed' send a notification to http://localhost:5050/notify (where Cygnus will be listening) with the 'speed' and 'oil_level' values. This subscription will have a duration of one month, and please, do not send me notifications more than one per second</i>.
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/commit/5b6a45eb7c12d4db2e56ac0147eb6851b51c3d9